sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,13 +1,16 @@
 # Release History
 
-## 12.2.0 (Unreleased)
+## 12.2.0 (2020-09-08)
 
 - Added RehydratePriority to BlobProperties and BlobItemProperties.
-- Fixed `BlockBlobClient.uploadStream()` to support bufferSize larger than `buffer.constants.MAX_LENGTH` for Jumbo Blob (~200TB Block Blob).
+- Fixed `BlockBlobClient.uploadStream()` to support `bufferSize` larger than `buffer.constants.MAX_LENGTH`.
 - Added support for Object Replication Service.
 - Added custom domain support.
 - Supported Append Blob Seal.
 - Supported tags conditional operations.
+- The Static Website Service now supports a DefaultIndexDocumentPath for a global HTTP 200 route within the static website. You can get it by `BlobServiceClient.getProperties()` and set it via `blobServiceClient.setProperties()`.
```

Comment:
any change in sdk for `DefaultIndexDocumentPath` or it is just service feature?

Reply:
Just added a new filed to the request and response of `BlobServiceClient.getProperties/setProperties`. Since the swagger change is taken in last preview release, this release only added a test case.